arlo_server/models.py

```diff
@@ -183,17 +186,32 @@ class ContestChoice(db.Model):
     )
 
 
-class ContestJurisdiction(db.Model):
-    contest_id = db.Column(
-        db.String(200), db.ForeignKey("contest.id", ondelete="cascade"), nullable=False,
-    )
-    jurisdiction_id = db.Column(
+contest_jurisdiction = db.Table(
+    "contest_jurisdiction",
+    db.Column(
+        "contest_id",
         db.String(200),
-        db.ForeignKey("jurisdiction.id", ondelete="cascade"),
+        db.ForeignKey("contest.id", primary_key=True, ondelete="cascade"),
         nullable=False,
-    )
+    ),
+    db.Column(
+        "jurisdiction_id",
+        db.String(200),
+        db.ForeignKey("jurisdiction.id", primary_key=True, ondelete="cascade"),
+        nullable=False,
+    ),
+)
```

Comment:
This is the recommended way to set up the relationship according to the flask-sqlalchemy docs. I'm a bit concerned about performance, because we have to look up all the jurisdictions when we set the `jurisdiction_ids` field on the contests.

https://flask-sqlalchemy.palletsprojects.com/en/2.x/models/#many-to-many-relationships

Reply:
not at all worried, that field will be auto-indexed and the lookups will be fast, plus you're already doing it in one query for all jurisdictions.  We'll want to review indexes at some point, but architecturally, this is sound.

Reply:
actually, the thing to be potentially worried about is not the query, but the *inserts* into the mapping table. We'll almost certainly want to background this at some point, cause it could be ~1600 inserts in a single contest. But that doesn't have to be in this PR.